benchmarks/time/src/main/scala/strawman/collection/immutable/HashSetBenchmark.scala

```diff
@@ -15,43 +15,40 @@ import scala.Predef.intWrapper
 @Measurement(iterations = 12)
 @State(Scope.Benchmark)
 class HashSetBenchmark {
-
   @Param(scala.Array(/*"0", */"1", "2", "3", "4", "7", "8", "15", "16", "17", "39", "282", "73121", "7312102"))
   var size: Int = _
 
   var xs: HashSet[Long] = _
-  var xss: scala.Array[HashSet[Long]] = _
   var zipped: HashSet[(Long, Long)] = _
   var randomIndices: scala.Array[Int] = _
-  var randomIndices2: scala.Array[Int] = _
-  var randomXss: scala.Array[HashSet[Long]] = _
+  def fresh(n: Int) = HashSet((1 to size).map(_.toLong): _*)
 
   @Setup(Level.Trial)
   def initData(): Unit = {
-    def freshCollection() = HashSet((1 to size).map(_.toLong): _*)
-    xs = freshCollection()
-    xss = scala.Array.fill(1000)(freshCollection())
+    xs = fresh(size)
     zipped = xs.map(x => (x, x))
     if (size > 0) {
       randomIndices = scala.Array.fill(1000)(scala.util.Random.nextInt(size))
-      randomIndices2 = scala.Array.fill(1000)(scala.util.Random.nextInt(size))
-      randomXss = scala.Array.fill(1000)(freshCollection().take(scala.util.Random.nextInt(size)))
     }
   }
 
   @Benchmark
+  @OperationsPerInvocation(1000)
   def incl(bh: Blackhole): Unit = {
-    var ys = HashSet.empty[Long]
+    var ys = fresh(size)
```

Comment:
I think we should not build a new collection within the benchmark. What do @Ichoran or @DarkDimius think?